src/DynamoCoreWpf/ViewModels/Search/NodeAutoCompleteSearchViewModel.cs

```diff
@@ -614,10 +614,9 @@ private NodeSearchElementViewModel GetViewModelForNodeSearchElement(NodeSearchEl
         /// </summary>
         /// <returns> Returns a list with a maximum MaxNumSearchResults elements.</returns>
         /// <param name="search"> The search query </param>
-        /// <param name="useLucene"> Temporary flag that will be used for searching using Lucene.NET </param>
-        internal IEnumerable<NodeSearchElementViewModel> SearchNodeAutocomplete(string search, bool useLucene)
+        internal IEnumerable<NodeSearchElementViewModel> SearchNodeAutocomplete(string search)
         {
-            if (useLucene)
+            if (LuceneSearchUtility != null)
```

Comment:
here the validation is correct? or should using LuceneSearchUtilityNodeAutocomplete != null?

Reply:
yes, changing it.